cli/api/client.go

```diff
@@ -338,7 +338,7 @@ func (c *Client) WriteOutputFile(kvb stores.KVBytes) {
 		os.Exit(1)
 	}
 
-	err = ioutil.WriteFile(c.config.Watcher.OutputPath, bts, 0644)
+	err = ioutil2.WriteFileAtomic(c.config.Watcher.OutputPath, bts, 0644)
```

Comment:
👍

Reply:
Do we know if non-Go clients depend on the modify-in-place behavior (and don't listen on `Create` events)?

Reply:
I checked the node.js and PHP clients--they seem fine.